### PR TITLE
Fix typos in Learn > Casacde layers

### PR DESCRIPTION
**Pull request by Arvton**
### Description

While using the guide to learn web development, I noticed some typos. Here they are:

Line 341: declared. layers. REMOVED extra period
Line 367: "precedence or normal styles" CHANGED TO "precedence for important styles"
Line 369: "precedence or important styles" CHANGED TO "precedence for important styles"

### Motivation

Submitting these changes to improve the readability of this article.

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 1]
```diff
@@ -338,7 +338,7 @@ Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Gl
 
 Animating styles have higher precedence than all normal styles, including inline normal styles.
 
-Important styles—property values that include the `!important` flag—take precedence over any styles previously mentioned on our list. They are sorted in reverse order of normal styles. Any important styles declared outside of a layer have less precedence than those declared within a layer. Important styles found within layers are also sorted in order of layer creation. For important styles, the last created layer has the lowest precedence, and the first created layer has the highest precedence among declared. layers.
+Important styles—property values that include the `!important` flag—take precedence over any styles previously mentioned on our list. They are sorted in reverse order of normal styles. Any important styles declared outside of a layer have less precedence than those declared within a layer. Important styles found within layers are also sorted in order of layer creation. For important styles, the last created layer has the lowest precedence, and the first created layer has the highest precedence among declared layers.
```
Comment (dipikabh) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:341]:
```suggestion
Important styles, that is, property values that include the `!important` flag, take precedence over any styles previously mentioned in our list. They are sorted in reverse order of normal styles. Any important styles declared outside of a layer have less precedence than those declared within a layer. Important styles found within layers are also sorted in order of layer creation. For important styles, the last created layer has the lowest precedence, and the first created layer has the highest precedence among declared layers.
```

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md [hunk 2]
```diff
@@ -364,9 +364,9 @@ To summarize:
 
 - The order of layers is the order in which the layers are created.
```
Comment (dipikabh) [inline files/en-us/learn/css/building_blocks/cascade_layers/index.md:365]:
```suggestion
- The order of precedence of layers is the order in which the layers are created.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thank you @Arvton for submitting these corrections!

I have proposed a few more edits to improve readability/tech accuracy.

File: files/en-us/learn/css/building_blocks/cascade_layers/index.md (hunk 1 above)
Comment (Arvton) [reply files/en-us/learn/css/building_blocks/cascade_layers/index.md:341]: I like it.

**Comment by Arvton** [comment]
Thank you for reviewing my PR. This is actually my first one for an open-source project. 😃

**Review by dipikabh — APPROVED** [review]

**Comment by Arvton** [comment]
Thanks. I want to get into a good rhythm with making contributions so I'll have a look at it.

**Pull request merged**